.gitignore

```diff
@@ -103,6 +103,7 @@ node_modules/
 /core-web/karma_html
 /core-web/installs
 /core-web/target
+/core-web/~/dotcms/*
```

Comment:
Is this change needed?

Reply:
@rjvelazco Yes, Talking with @fmontes we want to keep this one in case it happens to anyone else.

Reply:
@manuelrojas  Sounds god!